test/decoders/test_ops.py

```diff
@@ -824,6 +824,32 @@ def get_reference_frames(start_seconds, stop_seconds):
             frames, get_reference_frames(start_seconds, stop_seconds)
         )
 
+    @pytest.mark.parametrize("asset", (NASA_AUDIO, NASA_AUDIO_MP3))
```

Comment:
This test is from https://github.com/pytorch/torchcodec/pull/554 (which we should merge first)